pyproject.toml

```diff
@@ -26,7 +26,7 @@ docs = [
 ]
 
 [project.urls]
-repository = "magiepy.readthedocs.io"
+repository = "https://magie.ie"
```

Comment:
I think you want this to be the git repo no the data repo /web page but you can add other "urls" too e.g. https://github.com/sunpy/sunpy/blob/670439070c76f502c2eb6c68f0e2306720fa2686/pyproject.toml#L136-L142

Reply:
grand, put it as 

```
[project.urls]
repository = "https://github.com/TCDSolar/MagIEpy"
```